packages/cli/src/__tests__/ceramic-daemon.test.ts

```diff
@@ -51,6 +51,19 @@ const makeCeramicCore = async (ipfs: IpfsApi, stateStoreDirectory: string): Prom
   return core
 }
 
+async function makeDaemon(core: Ceramic, allowHistorical = true): Promise<CeramicDaemon> {
+  const port = await getPort()
+  const daemon = new CeramicDaemon(
+    core,
+    DaemonConfig.fromObject({
+      'http-api': { port },
+      indexing: { 'allow-queries-before-historical-sync': allowHistorical },
```

Comment:
if `allowHistorical` is false can we remove the `indexing` section entirely, to test the default behavior?